server/src/main/resources/reference.conf

```diff
@@ -1,4 +1,16 @@
 broccoli {
   nomad.url = "http://localhost:4646"
   templates.path = "./templates"
+  instances {
+    polling.frequency = 1
+    storage {
+      type = "fs"
+      fs.url = "./instances"
+      couchdb {
+        url = "http://localhost:5984"
+        dbName = "broccoli_instances"
+      }
+    }
+    parameters.defaultType = "string"
+  }
```

Comment:
I've added the new parameter to the README. I will update wiki after merging the PR and before closing the issue. Does it make sense?